### PR TITLE
update smart_proxy_dhcp_device42 to 1.0.7

### PR DESCRIPTION
**Pull request by tolyadouble**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.17
* [ ] 1.16
* [ ] 1.15

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

**Comment by tolyadouble** [comment]
@ekohl  I've updated repo with config, but it still raises the error.

**Comment by ekohl** [comment]
@tolyadouble you should change the `mv` command to move `.yml.example`. With that I expect it to work.

**Comment by tolyadouble** [comment]
@ekohl thanks, all good now

File: packages/plugins/rubygem-smart_proxy_dhcp_device42/rubygem-smart_proxy_dhcp_device42.spec [hunk 1]
```diff
@@ -85,6 +85,8 @@ mv %{buildroot}%{gem_instdir}/config/dhcp_device42.yml \
 %doc %{gem_instdir}/README.md
 
 %changelog
+* Mon Apr 23 2018 Anatolii Chmykhalo <anatolii.chmykhalo@device42.com>> 1.0.7-1
```
Comment (ekohl) [inline packages/plugins/rubygem-smart_proxy_dhcp_device42/rubygem-smart_proxy_dhcp_device42.spec]: There appears to be an additional `>` at the end

**Comment by tolyadouble** [comment]
@ekohl done

**Comment by tolyadouble** [comment]
Thanks, can we also merge it into 1.17?

**Pull request merged**